src/screens/DomainScreen/elements/RenderItemHeader.js

```diff
@@ -18,163 +11,172 @@ import MemoUnfollowDomain from '../../../assets/icon/IconUnfollowDomain';
-    )
-}
+          </GlobalButton>
+        ) : (
+          <GlobalButton buttonStyle={styles.noPh} onPress={handleFollow}>
+            <View style={styles.wrapperText}>
+              <MemoFollowDomain />
+            </View>
+          </GlobalButton>
+        )}
+      </View>
+    </View>
+  );
+};
 
 const styles = StyleSheet.create({
-    container: {
-        flexDirection: 'row',
-        paddingHorizontal: 20,
-        alignItems: 'center',
-        borderBottomWidth: 0.5,
-        borderBottomColor: COLORS.gray1,
-        paddingBottom: 8,
-        paddingTop: 8,
-    },
-    domainNameContainer: { flex: 1, marginRight: 8, },
-    wrapperImage: {
-        borderRadius: normalize(45),
-        borderWidth: 0.2,
-        borderColor: 'rgba(0,0,0,0.5)',
-        width: normalize(48),
-        height: normalize(48),
-        justifyContent: 'center',
-        alignItems: 'center',
-    },
-    image: {
-        height: normalize(48),
-        width: normalize(48),
-        borderRadius: normalize(45),
-    },
-    headerDateContainer: { flexDirection: 'row', alignItems: 'center' },
-    headerDomainName: {
-        fontSize: normalizeFontSize(14),
-        fontFamily: fonts.inter[600],
-        lineHeight: normalizeFontSize(16.9),
-        color: '#000000',
-    },
-    headerDomainDate: {
-        fontFamily: fonts.inter[400],
-        fontSize: normalizeFontSize(12),
-        lineHeight: normalizeFontSize(18),
-        color: '#828282',
-        flexShrink: 1,
-    },
-    headerFollowerText : {
-        color: '#828282',
-        fontSize: normalizeFontSize(12),
-        fontFamily: fonts.inter[700],
-    },
-    point: {
-        width: 2,
-        height: 2,
-        borderRadius: 4,
-        backgroundColor: colors.gray,
-        marginLeft: 5,
-        marginRight: 5,
-    },
-    domainItemTitle: {
-        fontSize: normalizeFontSize(16),
-        fontFamily: fonts.inter[700],
-        lineHeight: normalizeFontSize(24),
-    },
-    domainItemDescription: {
-        fontFamily: fonts.inter[400],
-        fontSize: normalizeFontSize(16),
-        lineHeight: normalizeFontSize(24),
-    },
-    domainIndicatorContainer: {
-        marginLeft: -4,
-        justifyContent: 'flex-start',
-    },
-    wrapperTextUnFollow: {
-        backgroundColor: '#00ADB5',
-        borderRadius: 8,
-        borderColor: '#00ADB5',
-        width: 28,
-        height: 28,
-        justifyContent: 'center',
-        alignItems: 'center',
-        borderWidth: 2,
-    },
-    wrapperText: {
-        backgroundColor: 'white',
-        borderRadius: 8,
-        borderColor: '#00ADB5',
-        width: 28,
-        height: 28,
-        justifyContent: 'center',
-        alignItems: 'center',
-        borderWidth: 2,
-    },
-    noPh: {
-        paddingHorizontal: 0
-    }
-})
+  container: {
+    flexDirection: 'row',
+    paddingHorizontal: 20,
+    alignItems: 'center',
+    borderBottomWidth: 0.5,
+    borderBottomColor: COLORS.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value for `borderBottomColor` has been updated. This change will affect the visual appearance of the component but should not impact its functionality.
```diff
- borderBottomColor: colors.gray1,
+ borderBottomColor: COLORS.gray1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->